apps/wpcom-block-editor/src/wpcom/features/redirect-onboarding-user-after-publishing-post.js

```diff
@@ -13,10 +12,11 @@ export function redirectOnboardingUserAfterPublishingPost() {
 	const siteSlug = window.location.hostname;
 
 	const unsubscribe = subscribe( () => {
+		const isSavingPost = select( 'core/editor' ).isSavingPost();
 		const isCurrentPostPublished = select( 'core/editor' ).isCurrentPostPublished();
 		const getCurrentPostRevisionsCount = select( 'core/editor' ).getCurrentPostRevisionsCount();
 
-		if ( isCurrentPostPublished && getCurrentPostRevisionsCount === 1 ) {
+		if ( false === isSavingPost && isCurrentPostPublished && getCurrentPostRevisionsCount === 1 ) {
```

Comment:
Why not using this instead?
```suggestion
		if ( ! isSavingPost && isCurrentPostPublished && getCurrentPostRevisionsCount === 1 ) {
```

Reply:
Yeah, I was ensuring the return type is false, but looking at the source [here](https://github.com/paulopmt1/gutenberg/blob/trunk/packages/editor/src/store/selectors.js#L687) and [here](https://github.com/paulopmt1/gutenberg/blob/trunk/packages/core-data/src/selectors.ts#L824) the return type will always be boolean. Thanks for the suggestion!